src/GRA.Domain.Service/AvatarService.cs

```diff
@@ -500,6 +516,7 @@ public async Task UpdateUserAvatarAsync(ICollection<AvatarLayer> selectionLayers
         public async Task<AvatarBundle> GetBundleByIdAsync(int id, bool includeDeleted = false)
         {
             var bundle = await _avatarBundleRepository.GetByIdAsync(id, includeDeleted);
+            bundle.AvatarItems = await _avatarItemRepository.GetBundleItemsAsync(id);
```

Comment:
I think AvatarItems are already being set in the bundle through AutoMapper and we don't need to do an additional call to set them.

Reply:
Fixed in commit 3fd2916ba9d6500015a137fc3b7ceebcca5cd5f5